File: wellness/UI/desktop/lib/models/treatment.dart
import 'package:json_annotation/json_annotation.dart';

part 'treatment.g.dart';

@JsonSerializable()
class Treatment {
  final int id;
  final String name;
  final String treatmentType;
  final String category;
  final String description;
  final double price;
  final int duration;
  final String picture;
  final int treatmentTypeId;
  final int categoryId;

  factory Treatment.fromJson(Map<String, dynamic> json) =>
      _$TreatmentFromJson(json);

  Treatment(this.id, this.treatmentType, this.category, this.description,
      this.price, this.picture, this.duration, this.name, this.treatmentTypeId, this.categoryId);

  Map<String, dynamic> toJson() => _$TreatmentToJson(this);
}

File: wellness/wellness.Model/Treatment/Treatment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace wellness.Model.Treatment
{
    public class Treatment
    {
        public int Id { get; set; }

        public string Name { get; set; } = string.Empty;
        public string TreatmentType { get; set; } = string.Empty;

        public string Category { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;
        public int Duration { get; set; }

        public float Price { get; set; }

        public byte[] Picture { get; set; } = null!;
        public int TreatmentTypeId { get; set; }

        public int CategoryId { get; set; }
    }
}
